src/components/screens/Authenticator/Authenticator.container.vue

```diff
@@ -141,6 +154,14 @@ export default {
       });
     },
 
+    openRoute(name) {
+      this.$router.push({ name }).catch(() => {});
```

Comment:
May be redirect to abstract 404 screen or route screen on error? I guess, it should handle cases when we are trying to navigate to non-existing route, right?

Reply:
nope, the problem with `.catch` is when you are make `.push` on the same route, `catch` excepts =(

Reply:
Mark with comment, it is not obvious now